src/pages/test-report/cat-a-mod1/components/speed-check/speed-check.ts

```diff
@@ -285,23 +285,27 @@ export class SpeedCheckComponent {
     return this.outcome === CompetencyOutcome.D;
   }
 
-  onFirstAttemptChange = (attemptedSpeed: number): void => {
+  onFirstAttemptChange = (attemptedSpeed: string): void => {
+    const firstAttempt = attemptedSpeed === '' ? undefined : Number(attemptedSpeed);
```

Comment:
@appleabs here is the line which will solve the problem that you are facing.

Reply:
Ah ok thanks!